views/components/news.njk

```diff
@@ -13,3 +13,28 @@
         </p>
     </article>
 {% endmacro %}
+
+{% macro latestNews(articles) %}
+    <div class="latest-news">
+        <div class="latest-news__header">
+            <h3 class="t1 t--underline a--text accent--blue">
+                <a class="u-link-minimal" href="{{ buildUrl("uk-wide/news-and-events") }}">
+                    {{ __('global.misc.latestNews') }}</h3>
+                </a>
```

Comment:
Missing a closing `</h3>` tag here?

Reply:
Good catch

Reply:
(This has been corrected, but the comment is on the `</a>` so the comment isn't dismissed.